meeting_minutes/211028_SARIF_TC_47.md

```diff
@@ -0,0 +1,148 @@
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+* Approved Teleconferences (Tuesdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+  ```
+  November 11
+  December 2
+  ```
+* Possible face-to-face meeting in 2022Q1
+
+# 3. Discussion
+
+## 3.1 Review recruitment effort to complete the technical committee
+
+* Michael: New members in sight
+* Michael: Reached out towards StackHawk, ForAllSecure, and NTT in the dynamic analyisa area
+* Katrina: Contacted ex-colleague (now at salesforce) in dynamic analysis field and others
+
+## 3.2 Review status on finalizing SARIF 2.1 errata
+
+* Michael: DONE and would like to send the change bar documents out per a branch so interested reviewers can review there
+* David: Suggests to directly create a branch and send an email with the URL
+* Michael: Will store in microsoft OneDrive and manage access - so as to make comments visible (as the document is a Microsoft Word document no markdown)
+* David: When this review is approved by the reviewers we should hold a full majority vote and hand over publication to OASIS TC administration.
+
+## 3.3 Review current state of ecosystem ongoing work
+
+* Michael: How about placing names in the chat.
+* Eddy:
+  - updated the [PR#2](https://github.com/dmk42/sarif-spec/pull/2) to add more tools to the wiki
+  - finished the change to enable SARIF for [AttackSurfaceAnalyzer](http://github.com/microsoft/attackSurfaceAnalyzer)
+  - Visual Studio support for suppressions management, will port this feature to VS Code as well.
+  - Updated the VS Studio extension to 64-bit
+  - working with BPA ([Template Analyzer](https://github.com/Azure/template-analyzer)) and TM7 (log file for the Microsoft Threat Modeling)
+* David: Merged the [PR#2](https://github.com/dmk42/sarif-spec/pull/2)
+
+## 3.4 Discuss metrics
+
+* All discuss metrics representations
+* Michael suggests for next meeting to 
+  - keep agenda items 3.1, 3.2, and 3.3 and 
+  - change the item 3.4 to "Discuss metrics specifically CodeQL, Metrics Investments, and General design approach"
+
+## 3.5 Discuss dynamic analysis and how to address it
+
+* David: Reminds everyone what was last time decided to proceed in that order:
+  - publication of Errata for 2.1 
+  - rechartering of TC to widen scope
+  - there will be a specific list needed for the new charter
+* Michael: Let us start a collection of domains
+* David: Domain fuzzing
+* Michael: Domain test execution and colocate with code coverage
+* Paul: Domain crash dump analysis? Or too close to fuzzing?
+* Michael: Supports adding the crash dump analysis explicit
+* Katrina: Domain runtime analysis?
```

Comment:
I actually thought that runtime analysis should be out of scope -- perhaps that's what you meant by "?"...